internal/querycompute/querycompute.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+option go_package = "github.com/Microsoft/hcsshim/internal/querycompute;querycompute";
+
+import weak "gogoproto/gogo.proto";
+
+service QueryCompute {
+    rpc ComputeProcessorInfo(ComputeProcessorInfoRequest) returns (ComputeProcessorInfoResponse);
```

Comment:
This can be just `GetProcessorInfo` or `ProcessorInfo`. (Same for messages below)